main.py

```diff
@@ -66,7 +66,8 @@ def add_handlers(dp: Dispatcher):
     dp.add_handler(CommandHandler("settings", settings_cmd))
     dp.add_handler(CommandHandler("sban", sban_cmd))
     dp.add_handler(CommandHandler("clean_pending", clean_pending_cmd))
-    dp.add_handler(CommandHandler("purge", purge_cmd, run_async=True))
+    dp.add_handler(CommandHandler("db_backup", db_backup_cmd))
+    dp.add_handler(CommandHandler("purge", purge_cmd))
```

Comment:
In the /purge command, missing
```python
run_async=True
```

Reply:
You could also add it to the /db_backup command: Maybe test it before adding it

Reply:
ups